source/_integrations/androidtv.markdown

```diff
@@ -76,6 +76,11 @@ get_sources:
   required: false
   default: true
   type: boolean
+source_provider:
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
This is not yet supported on the androidtv component and is a separate task to move it over imo.

```
androidtv duffy$ grep -R "ConfigFlow" . | wc -l
0
```

Reply:
That is correct, but that is the current state of the architectural decisions made.

Reply:
Will close the PR then I guess?